libraries/Microsoft.Bot.Builder.AI.QnA/Utils/GenerateAnswerUtils.cs

```diff
@@ -51,7 +51,7 @@ public GenerateAnswerUtils(IBotTelemetryClient telemetryClient, QnAMakerEndpoint
         /// <param name="messageActivity">Message activity of the turn context.</param>
         /// <param name="options">The options for the QnA Maker knowledge base. If null, constructor option is used for this instance.</param>
         /// <returns>A list of answers for the user query, sorted in decreasing order of ranking score.</returns>
-        public async Task<QueryResult[]> GetAnswersAsync(ITurnContext turnContext, IMessageActivity messageActivity, QnAMakerOptions options)
+        public async Task<QueryResults> GetAnswersAsync(ITurnContext turnContext, IMessageActivity messageActivity, QnAMakerOptions options)
```

Comment:
Can't make this change due to compatablity concerns. Leave this old method in place, and create a new one. 

Public method / public class....

Reply:
This methods is not used anywhere. Everyone calls GetAnswersAsync method present in QnAMaker.cs file.

Reply:
This is a public method on a public class, which means compat needs to be preserved. That's a hard requirement that we've got.